### PR TITLE
prs

### PR DESCRIPTION
**Pull request by denismwangi**
added main method and few functionalities

### CODE REVIEW

**Comment by Ericgacoki** [comment]
@denismwangi, thank you for creating your first PR in this repository. Every PR goes through a technical review before merging.

File: Car game.py [hunk 1]
```diff
@@ -1,42 +1,74 @@
...
-            else: #this is executed when the player enters un understood action. ie action that is not accepted
-                print("Sorry Boss, I don't understand that...\n") #the car doesn't know what you mean. At this point,you can ask for help 
-        else: #when the 15 chances are depleted
-            print("GAME COMPLETED !.")      # game is over
-            print(name+"'s","score = $ 150.21 .Congrats !! ") #this is just to entertain the player and to prompt him/her to play again. This value is fixed
-    else: #if player selects a car from nowhere. ie car not in the list of available cars.
-        print("INVALID CAR SELECTION. ",name," PLEASE TRY AGAIN") #alert for an invalid selection and 
-        break                                                                                      # the game automatically exits.
+from display import help_screen
+
```
Comment (Ericgacoki) [inline Car game.py:2]: please remove unnecessary indents

**Review by Ericgacoki — CHANGES_REQUESTED** [review]

File: Car game.py (hunk 1 above)
Comment (denismwangi) [reply Car game.py:2]: it well indented since i implemented classes and method

**Review by denismwangi — COMMENTED** [review]
done!

**Review by Ericgacoki — APPROVED** [review]

**Pull request merged**